packages/core/lib/src/internal/ops/tag_table.dart

```diff
@@ -244,14 +243,14 @@ class TagTable {
 extension _BuildMetadataExtension on BuildMetadata {
   static final _rows = Expando<int>();
 
-  set row(int v) => _rows[this] = v;
-  int get row => _rows[this];
+  set row(int? v) => _rows[this] = v;
```

Comment:
```suggestion
  set row(int v) => _rows[this] = v;
```

Reply:
I had to make the getter not nullable as well because the getter always has to be of the same type as the setter. It doesn't seem to cause any problem.